pycognito/__init__.py

```diff
@@ -260,6 +260,7 @@ def verify_token(self, token, id_name, token_use):
                 issuer=self.user_pool_url,
                 options={
                     "require": required_claims,
+                    "verify_iat": False,
```

Comment:
This one makes sense as the range was not validated before.
However, disabling this will also disable the check to see if it is an integer. That part should still be verified.

Reply:
Made an alternative with that here <https://github.com/NabuCasa/pycognito/pull/252>